libminifi/include/utils/StringUtils.h

```diff
@@ -505,7 +505,7 @@ class StringUtils {
    */
   static bool matchesSequence(std::string_view str, const std::vector<std::string>& patterns);
 
- private:
+  static bool splitToUnitAndValue(std::string_view input, std::string& unit, int64_t& value);
```

Comment:
can we also switch the arguments, please?
```suggestion
  static bool splitToUnitAndValue(std::string_view input, int64_t& value, std::string& unit);
```
that would look more logical to me

Reply:
sure thing, I've changed the name aswell to better reflect the order of the arguments in https://github.com/apache/nifi-minifi-cpp/pull/1543/commits/64c5975e3fda0911820465778fa4d73f16d12002